engine/audio/src/audiocapture.cpp

```diff
@@ -227,12 +224,15 @@ void AudioCapture::processData()
 #endif
 
     // 5 ********* Calculate the average signal power
-    m_signalPower = pwrSum / m_captureSize;
-
-    // 6 ********* Calculate vector magnitude
     foreach(int barsNumber, m_fftMagnitudeMap.keys())
     {
-        double maxMagnitude = fillBandsData(barsNumber);
+        maxMagnitude = fillBandsData(barsNumber);
+        pwrSum = 0.;
+        for (int n = 0; n < barsNumber; n++)
+        {
+            pwrSum += m_fftMagnitudeMap[barsNumber].m_fftMagnitudeBuffer[n];
+        }
+        m_signalPower = pwrSum * 500. / barsNumber;
```

Comment:
The 500 here is a magic value for calibrating the maximum value of the graph to the power values.  However I still don't know the scale of the graph so I am purely guessing here.